src/Arcus.EventGrid.Testing/Infrastructure/Hosts/EventConsumerHost.cs

```diff
@@ -108,5 +108,13 @@ private string TryGetReceivedEvent(string eventId)
 
             return rawEvent;
         }
+
+        private static string DetermineEventId(JToken parsedEvent)
+        {
+            Guard.NotNull(parsedEvent, nameof(parsedEvent));
+
+            var eventIdNode = ((JObject)parsedEvent).GetValue("Id", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Isn't there a `.TryGetValue`? Maybe that's more clear than syntactic sugar `?`

Reply:
Good point, updated!